lang/java/avro/src/main/java/org/apache/avro/io/BinaryData.java

```diff
@@ -181,16 +184,7 @@ private static int compare(Decoders d, Schema schema) throws IOException {
    * return a positive value, if less than return a negative value.
    */
   public static int compareBytes(byte[] b1, int s1, int l1, byte[] b2, int s2, int l2) {
```

Comment:
This method could be marked as deprecated for 1.12 and removed in 1.13.0

Reply:
OK. Will try to follow up, but I always worry about pulling back stuff and then adding any kind of friction to consumers to upgrade.